crates/re_types/definitions/rerun/blueprint/archetypes/background.fbs

```diff
@@ -10,19 +10,20 @@ namespace rerun.blueprint.archetypes;
 
 // ---
 
-/// Configuration for the background of the 3D space view.
-table Background3D (
+/// Configuration for the background of a space view.
```

Comment:
```suggestion
/// Configuration for the background of a view.
```
We started dropping the "space" part from blueprint apis (similarly it's already a TimeSeriesView, not a TimeSeriesSpaceView)